app/Console/Commands/ImportSignupsCommand.php

```diff
@@ -70,13 +73,17 @@ public function handle()
                     'northstar_id' => $missing_signup['northstar_id'],
                     'campaign_id' => $missing_signup['campaign_node_id'],
                     'campaign_run_id' => $missing_signup['campaign_run_id'],
-                    'source' => 'sms',
-                    'created_at' => $missing_signup['signup_created_at_timestamp'],
+                    'source' => 'phoenix-next',
+                    'created_at' => $missing_signup['signup_created_at_timestamp'] ? $missing_signup['signup_created_at_timestamp'] : Carbon::now(),
```

Comment:
If this is just doing an insert into the `signups` table, it should set a `created_at` automatically, right? I don't think we will need to set it manually as `now()`

Reply:
@sbsmith86 that is correct but we still allow the option to backdate so we need to be able to set the `created_at` if provided

Reply:
@katiecrane if we do something like @DFurnes outlined, I think that will allow us to either add the column for backdating or not.

Reply:
@sbsmith86 @DFurnes I went the quick and less elegant route since I think we need to run this today. Would you both be okay holding off on these updates until next time we need to import signups?